src/dbnode/client/session_test.go

```diff
@@ -621,9 +626,9 @@ func testHealthCheck(err error, bootstrappedNodesOnly bool) func(rpc.TChanNode,
 }
 
 func noopNewConnection(
-	channelName string,
+	_ string,
 	addr string,
-	opts Options,
+	_ Options,
 ) (Channel, rpc.TChanNode, error) {
-	return &noopPooledChannel{addr}, nil, nil
+	return &noopPooledChannel{addr, 0}, nil, nil
```

Comment:
Nit: no need to init a 0 value.

Reply:
when no field names are used its mandatory